ui/desktop/CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## V2.2.0 [CLI 0.18.0] (2024.10.14)
+
+### New
+
+- Upgrade CLI to v0.18.0 ([PR](https://github.com/hashicorp/boundary-ui/pull/2516))
+- Added support for notifications with transparent sessions ([PR](https://github.com/hashicorp/boundary-ui/pull/2336))
+- Added a user settings page for configuration options ([PR](https://github.com/hashicorp/boundary-ui/pull/2499))
+- Changed the limit of shown results to 250 and an indicator when cache daemon is loading ([PR](https://github.com/hashicorp/boundary-ui/pull/2500))
+
+### Bug Fixes
+
+- Fix to show username credentials before password ([PR](https://github.com/hashicorp/boundary-ui/pull/2494))
```

Comment:
Is this a bug fix? Seems like this would go under new as an improvement.

Reply:
🤷 I guess it depends on how you view the original intent, I'll move it over